pkg/app/pipedv1/plugin/kubernetes/deployment/rollback_test.go

```diff
@@ -88,21 +90,23 @@ func TestPlugin_executeK8sRollbackStage_SuccessfulRollback(t *testing.T) {
 	dtConfig, dynamicClient := setupTestDeployTargetConfigAndDynamicClient(t)
 
 	// Read the application config from the example file
-	cfg, err := os.ReadFile(filepath.Join(examplesDir(), "kubernetes", "simple", "app.pipecd.yaml"))
+	cfgBytes, err := os.ReadFile(filepath.Join(examplesDir(), "kubernetes", "simple", "app.pipecd.yaml"))
+	require.NoError(t, err)
+	cfg, err := sdk.DecodeYAML[kubeConfigPkg.KubernetesApplicationSpec](cfgBytes)
```

Comment:
[imo] 
It would be nice to define a helper function to create `ApplicationConfigSpec` from a file or string.
I feel users don't need to be aware of `sdk.DecodeYAML` in the actual logic.
WDYT?

Reply:
I agree with you, so I added sdk.LoadApplicationConfig and removed sdk.DecodeYAML
https://github.com/pipe-cd/pipecd/pull/5740/commits/5abbdd5f8388fb414ba8c3187421fcedd9e75150

Reply:
@Warashi 
Thank you for adding the helper method!
I read the comment and think that we can make a test package `sdk/sdktest` . How about you?
```
/ LoadApplicationConfig loads the application config from the given filename.
// This is intended to use in the tests.
```

Reply:
@ffjlabo 

It sounds good.
How about making the `sdktest.LoadApplicationConfig` 's first argument `t *testing.T` and automatically calling `t.Fatal` when it meets an unmarshalling error and making its return value only `*ApplicationConfig[Spec]`?

Reply:
@Warashi Sounds great!!! Let's go with that!

Reply:
Implemented and switched the usage.
https://github.com/pipe-cd/pipecd/pull/5740/commits/30bd7531d3822d7ccabbd505e0bb70fbe401317c
https://github.com/pipe-cd/pipecd/pull/5740/commits/4885a409247c713d687094d1e2ee6255f81c39ae

Reply:
Sounds nice! That's a nice improvement!